src/procedure/nodes/select.cpp

```diff
@@ -237,7 +237,7 @@ bool SelectProcedureNode::execute(ProcessPool &procPool, Configuration *cfg, std
                 continue;
 
             // Check Site exclusions
-            if (excludedSites_.contains(&site))
+            if (excludedSites_.end() != std::find(excludedSites_.begin(), excludedSites_.end(), &site))
```

Comment:
Again, reversing the compared quantities makes more sense to me:
```suggestion
            if (std::find(excludedSites_.begin(), excludedSites_.end(), &site) != excludedSites_.end())
```